doc/HOSTING

```diff
@@ -0,0 +1,116 @@
+# introduction
+
+This document tries to explain what is required to host a
+dedicated server, provided you already have a working
+unvanquished installation and "pak" files (those who have a
+`.dpk` file extension), which are not necessarily the
+official ones.
+
+# non-goals
+It does *not* tries to explain how to compile or modify
+unvanquised, how to configure a server with modified engine,
+how to setup your system or network, and aims at the
+simplest possible configuration.
+
+# pre-requisites
+
+First of all, you need a computer than can be reached by the
+Internet. A computer on your LAN might do the job, but you
+will need to configure your router beforehand.
+You will also need an HTTP server serving on port 80. While
+this is not strictly necessary, it is highly recommended.
+Not doing this means that if your players lack one of the
+requirements, they might be unable to join at all.
+I'll skip the details on this, as this discussion is out
+of the scope for this current document.
+
+# Daemon installation
+
+Create a directory in which you will place the following
+files:
+
+* daemonded
+* crash_server
+* nacl_helper_bootstrap
+* nacl_loader
+* irt_core-x86_64.nexe
+
+# HTTP configuration
+
+You will need to have a directory (I advise you to dedicate
+it to unvanquished, but this is not mandatory) containing
+`.dpk` files and another file named `PAKSERVER`, containing
+the string `ALLOW_UNRESTRICTED_DOWNLOAD`.
+If you get this wrong, your users will experience the
+infamous "UDP download", which is very slow. Slow as in,
+less than 30KiB/s, meaning most changes will litterally take
+hours to download.
+
+# starting the server
+
+To start the game, you need to run the command:
+
+```
+daemonded -homepath PATH_TO_CONFIG -pakpath PATH_TO_PAKS \
+	+exec server.cfg
+```
+
+PATH_TO_PAKS is the relative or absolute path from where
+you started the process to reach the HTTP directory you
+setup just before.
+
+PATH_TO_CONFIG is the relative or absolute path from where
+you started the process to reach the game's configuration
+and logs folder.
+
+`server.cfg` can be any file in `PATH_TO_CONFIG/config`
+folder, but it will be easier to get help if you stick with
+conventional names.
+
+## server "homepath" setup
+
+homepath will expect a specific layout. Files there are not
+technically required, but not having them will mean game
+will rely on the built-in alternatives.
+
+* `config/`
+* `config/server.cfg`
+* `config/map/`
```

Comment:
I would say that map specific configs definitely do not fall under "simplest possible set up". Also its confusing that "map" refers to something like "plat23" instead of the word "map"

Reply:
One thing is, the "map/default.cfg" file is really, really useful currently. Notably for bot names, or to have different number of humans/aliens bots.